addons/website/static/src/snippets/s_process_steps/options.js

```diff
@@ -81,7 +81,7 @@ options.registry.StepsConnector = options.Class.extend({
     _reloadConnectors() {
         const possibleTypes = this._requestUserValueWidgets('connector_type')[0].getMethodsParams().optionsPossibleValues.selectClass;
         const type = possibleTypes.find(possibleType => possibleType && this.$target[0].classList.contains(possibleType)) || '';
-        const steps = this.$target[0].querySelectorAll('.s_process_step');
+        const steps = this.$target[0].querySelectorAll('.s_process_step:not(.o_snippet_desktop_invisible)');
```

Comment:
I really have the same worries here: how can this work outside of edit mode?

Reply:
It's quite the same as the other one. As the connectors are only for the desktop view we can ignore steps that are only display in mobile view :+1: I will add a comment :)